src/entities/flanker/ui/NativeIosFlanker/NativeIosFlanker.tsx

```diff
@@ -31,18 +31,15 @@ const NativeIosFlanker: FC<Props> = props => {
 
     const configString = JSON.stringify(configuration);
 
-    NativeModules.FlankerViewManager.preloadGameImages(configString);
-
-    // Race condition issue:
-    // FlankerView.swift may be created after startGame call
-
-    setTimeout(() => {
-      NativeModules.FlankerViewManager.setGameParameters(configString);
-      NativeModules.FlankerViewManager.startGame(
-        props.configuration.isFirstPractice,
-        props.configuration.isLastTest,
-      );
-    }, 600);
+    NativeModules.FlankerViewManager.preloadGameImages(configString)
+      .then(() => {
+        NativeModules.FlankerViewManager.setGameParameters(configString);
+        NativeModules.FlankerViewManager.startGame(
+          props.configuration.isFirstPractice,
+          props.configuration.isLastTest,
+        );
+      })
+      .catch((error: string) => Logger.log(error));
```

Comment:
Please change the message to `Logger.log('[NativeModules.FlankerViewManager.preloadGameImages] An internal error occurred during caching images or starting the game: \n\n', error)`